### ISSUE TITLE
IoTCookBook: Brocken links on the documentation

### ISSUE THREAD

**Issue by Xyala** [crossbario/crossbar#796]
Noticed that all the links to the iotcookbook are broken on the main crossbar.io website.
I was working on the Yun and all of a sudden everything is gone. 

http://crossbar.io/iotcookbook/

Is there another place where I can find it?
Thanks.

Tim.

**Comment by oberstet** [comment]
@Xyala yeah, sorry, I am aware. It'll be online soon again. The reason is that we did some refactoring of "docs": Crossbar.io docs now live here, whereas the IoT cookbook has moved somewhere else. It is not a Crossbar.io issue, but I leave this open nevertheless in this case.

**Comment by Xyala** [comment]
OK, no problem thanks! I was able to find some of the things through the google cache - so I'm good for now.

I know it might be a bit off topic, but I ran the tutorial 3 out of the iotcookbook for the yun (https://github.com/crossbario/crossbarexamples/blob/master/iotcookbook/device/yun/tutorial/tutorial3/bridge.js ), and I'm getting some error when executing the js code. Something related to node and MIPS.

/usr/bin/nodejs: '/usr/lib/node_modules/autobahn/node_modules/ws/node_modules/bufferutil/build/Release/bufferutil.node' is not an ELF executable for MIPS
/usr/bin/nodejs: '/usr/lib/node_modules/autobahn/node_modules/ws/node_modules/utf-8-validate/build/Release/validation.node' is not an ELF executable for MIPS

Do you know if I did something wrong, or forgot a step somewhere?

**Comment by oberstet** [comment]
@Xyala rgd your issue .. likely: you have built node modules that contain native code, but that native code has been compiled for x86 (on your desktop), not for MIPS (what the Yun is running on).

**Comment by Xyala** [comment]
Hmm..I didn't compile anything. I only did things following the steps form the quick setup guide for the Yun. (did it twice actually with a full factory reset in between as it wasn't working - thought I might have missed something).

Which is weird because I used to have a yun with that working setup a few months ago. 
Is there something in the quick setup procedure that requires some specific attention to avoid this?
